bookkeeper-server/src/main/java/org/apache/bookkeeper/replication/Auditor.java

```diff
@@ -620,7 +622,7 @@ public void shutdown() {
         LOG.info("Shutting down auditor");
         executor.shutdown();
         try {
-            while (!executor.awaitTermination(30, TimeUnit.SECONDS)) {
+            while (!executor.awaitTermination(awaitTermination, TimeUnit.SECONDS)) {
```

Comment:
It will wait at most 30s before enter the shutdownNow logic, not wait at least 30s.